src/control/cmd/dmg/storage.go

```diff
@@ -78,6 +80,9 @@ func (cmd *storagePrepareCmd) Execute(args []string) error {
 	}
 
 	if prepScm {
+		if cmd.jsonOutputEnabled() && !cmd.Force {
+			return cmd.errorJSON(errors.New("Cannot use --json without --force"))
+		}
```

Comment:
Shouldn't we check this up at the top before we do anything else? Is there a reason why we wouldn't want to check this when prepping NVMe?

Reply:
Well, the main reason I put it in the prepScm block is that we only prompt for user confirmation when we're preparing SCM (due to the need to reboot).